### DISCUSSION TITLE
Bring back achievement rating

### DISCUSSION THREAD

**Discussion by kdecks** [RetroAchievements/RAWeb#663] [Ideas]
BUT only allow those who have mastered or completed the set to rate. Could lower the threshold to as low as 50% completed before you can rate. 

Also when a set is revised, reset the rating.

**Comment by julkasalsa** [comment]
Instead of giving the mimimum completion requirement I would prefer if the achievement rating would be embeded with comments (you need to comment to give the rating). I think rating should be declared and not definitive. The main goal of rating is be to bring information to the community about consumptional quality of the set. Leaving the voters classified don't give any advantage, and if there are no comments, the developer (or the community) don't know what to change about the set, or treat a given vote seriously.

**Comment by kdecks** [comment]
Perhaps a dedicated place on game pages for reviews. I think this ideas to be conceptualized more fully before its ready.

**Comment by Altomar79** [comment]
I would like the idea of a popup message on the site when the player reach half points on either softcore or hardcore mode (or mastering the set) asking him or forcing him to answer about his opinion for the set and depending on how he answer (with max of 5 points for example) if he vote for 3 or under to force him put a comment explaining what he didnt like so the developers can improve the set on the future.
